### ISSUE TITLE
Parsing error in detect.py, detect.py and utils.py

### ISSUE THREAD

**Issue by yashlad27** [theAIGuysCode/yolov4-custom-functions#107]
![image](https://github.com/theAIGuysCode/yolov4-custom-functions/assets/71868797/8abf734c-ef6f-4943-bc13-444fe7a8bf46)

**Comment by HawkingRadiation42** [comment]
yes im getting the same error, has anyone found a fix for it yet?

**Comment by albertwow80** [comment]
> yes im getting the same error, has anyone found a fix for it yet?

I downgrade opcv to 4.1.1.26 and solve this issue.

**Comment by roysG** [comment]
I get error:

ERROR: Could not find a version that satisfies the requirement opencv-python==4.1.1.26 (from versions: 3.4.0.14, 3.4.10.37, 3.4.11.39, 3.4.11.41, 3.4.11.43, 3.4.11.45, 3.4.13.47, 3.4.15.55, 3.4.16.57, 3.4.16.59, 3.4.17.61, 3.4.17.63, 3.4.18.65, 4.3.0.38, 4.4.0.40, 4.4.0.42, 4.4.0.44, 4.4.0.46, 4.5.1.48, 4.5.3.56, 4.5.4.58, 4.5.4.60, 4.5.5.62, 4.5.5.64, 4.6.0.66, 4.7.0.68, 4.7.0.72, 4.8.0.74, 4.8.0.76)
ERROR: No matching distribution found for opencv-python==4.1.1.26

How can i solve it?

@albertwow80
